tests/wallet/did_wallet/test_did_rpc.py

```diff
@@ -1,121 +1,57 @@
-import logging
+import asyncio
```

Comment:
So we have this file now to test some DID RPCs and we have a section in `test_wallet_rpc.py`, see https://github.com/Chia-Network/chia-blockchain/blob/a9b1ba8575edcf8eefe15751c24d5f77c6c14b56/tests/wallet/rpc/test_wallet_rpc.py#L758

I think this should be consolidated.

Reply:
@ytx1991 can you look into this?